### ISSUE TITLE
Fix spelling issues

### ISSUE THREAD

**Issue by TwistedTwigleg** [TwistedTwigleg/Godot_FPS_Tutorial#10]
There are several spelling issues in the scripts (comments, variable names, etc) that need fixing. (Probably, I have not checked just yet but some copied code had spelling issues). This will likely involve several commits to fix across all branches.

The documentation tutorial pages will also need to be updated to reflect the changes.

**Comment by TwistedTwigleg** [comment]
BTW: just because it is marked low priority does not mean it's not important. It's just I'd rather be working on adding stuff than looking around for (and cringing when I find) my spelling mistakes 😃

**Comment by TwistedTwigleg** [comment]
Mostly (if not completely) fixed in #16,

The documentation will be updated in the tutorial remaster.
(I'll have to redo most of the old branches, so I decided not to wait on them before closing this branch)
